Makefile

```diff
@@ -0,0 +1,13 @@
+init:
+	pipenv install -d
```

Comment:
Nitpicking, but if you used full-blown `--dev` I wouldn't have to read the docs for its meaning.

Reply:
Sorry for this, I have been using pipenv for almost a year now and `-d` came as second nature.